pgsql_auto_backup/i18n/bg.po

```diff
@@ -35,7 +35,7 @@ msgid "Invalid model name in the action definition."
 msgstr "Невалидно име на модел при задаване на действие"
 
 #. module: pgsql_auto_backup
-#: model:ir.model,name:auto_backup.model_db_backup
+#: model:ir.model,name:auto_backup.model_DbBackup
```

Comment:
I think this is a bad search/replace... you can undo your changes in the .po files